src/Compilers/Core/Portable/Operations/OperationNodes.cs

```diff
@@ -363,4 +363,20 @@ internal sealed partial class StaticLocalInitializationSemaphoreOperation
         {
         }
     }
+
+    internal sealed partial class BlockOperation
+    {
+        public static BlockOperation CreateTemporaryBlock(ImmutableArray<IOperation> statements, SemanticModel semanticModel, SyntaxNode syntax)
+            => new(statements, semanticModel, syntax);
+
+        private BlockOperation(ImmutableArray<IOperation> statements, SemanticModel semanticModel, SyntaxNode syntax)
+            : base(semanticModel, syntax, isImplicit: true)
+        {
+            // Intentionally skipping SetParentOperation: this is used by CreateTemporaryBlock for the purposes of the
```

Comment:
>Intentionally skipping SetParentOperation [](start = 15, length = 41)

What are the consequences of doing this? Certain things will not work/throw, etc. ?

Reply:
Added more detail to the comment.

---
In reply to: [526903983](https://github.com/dotnet/roslyn/pull/49453#discussion_r526903983) [](ancestors = 526903983)